README.md

```diff
@@ -51,54 +51,48 @@
 - NPM
 - Fetch API 
 - ESLint
+- Miro Wireframe [(link)](https://miro.com/app/board/o9J_lyZfIKA=/)
+- [PokeApi](https://pokeapi.co/)
 
-## Setup
+## Accessing and Using the App
 
-Fork this repo (look up in the top right corner) and copy the SSH link.
-Open your terminal, run git clone (replace the [ ] with the copied remote address)
-Change into the directory and run
+[deployment link](deployment link)
 
-```bash
-npm i
-```
-To view, run:
+[describe steps of how to use & navigate the app]
 
-```bash
-npm start
-```
+## GIFs
 
-or click the link:[Add deployed link here]
-
-## Illustrations
-
-- User can select a PoKemon Card, find more details about it .
+- User selects a Pokemon to find more details about it
 ![recording (8)]()
 
-
-- User can search for a particular PoKemon in the PoKedeX search bar.
+- User searches for a particular Pokemon in the PoKedeX search bar
 ![recording (9)]()
 
-- Fully responsive user experience.
+- Fully responsive user experience
 ![recording (11)]()
 
-## 🏆 Wins
+## Reflections
+
+### Wins
+
+<img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/7.svg" alt="Pikachu" width="100" align="right"/>
+
+- Creating the React components and have them communicate as desired
+- Learning the behavior of JSX and how to apply CSS to elements
+- Including a search bar for search Pokemon by name for better user experience
+- Applying Cypress testing
+- Receiving a Lighthouse score over 90, even while still learning JSX
 
-- A big win was to be able to create the React components and have them communicate as we wanted them to.
-- Learning the behavior of JSX and how to apply CSS to those elements. 
-- Including a search bar for search PoKemon By name for better user experience.
-- Being able to successfully apply the Cypress testing.
-- Having a Lighthouse score over 90, even as we were still getting used to using JSX.
+### Challenges
 
-## Challenges
+<img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/4.svg" alt="Pikachu" width="100" align="right"/>
 
 - Balancing fun/life with project/work/study time 👩🏻‍💻 💅🏻 💆🏻‍ 🐈 🎸 
-- First project using GSAP and PWA .
-- Managing the media queries for responsive design.
-- Navigating the challenge of learning two new tecnologies.
+- Managing the media queries for responsive design
+- Learning and using two new tecnologies at the same time (GSAP and PWA)
 
 ## Future Designs/Extensions
 
-- Add a user login/logout with username and password .
-- Have an area of favorited cards.
-- Be able to play the game in the app.
-- The User's favotiting will be saved within the user's account.
+- The user will be able to login/logout with username and password.
```

Comment:
Love the user story, I think it gives the read me a clear description of what the app is!